.gitattributes

```diff
@@ -8,6 +8,8 @@ go.sum -diff -merge linguist-generated=true
 *_easyjson.go -diff -merge
 *_easyjson.go linguist-generated=true
 pkg/security/probe/constantfetch/btfhub/constants.json -diff -merge linguist-generated=true
+# Fixtures should have LF line endings because they are checked against OCI packages built on Linux
+pkg/fleet/internal/fixtures/** text=auto eol=lf
```

Comment:
nit: would be better to specify only the text files, but git /probably/ won't think the `.tar` files are text files